website/src/components/canvas/Canvas.jsx

```diff
@@ -4,14 +4,24 @@ import FlatButton from 'material-ui/FlatButton';
 import Paper from 'material-ui/Paper';
 import Sidebar from './Sidebar';
 import TextField from 'material-ui/TextField';
-// import {List, ListItem} from 'material-ui/List/';
 import Avatar from 'material-ui/Avatar';
```

Comment:
These material UI imports should still be in order, so Avatar should be at the top right under the { Box } import from reflexbox